tests/test_spectral_cum.py

```diff
@@ -0,0 +1,39 @@
+import numpy as np
+import pytest
+import os
+
+import perturbopy.postproc as ppy
+
+
+@pytest.fixture()
+def sto_spectral_cum():
+    """
+    Method to generate the spectral_cum object corresponding to spectralhdf5.
+
+    Returns
+    -------
+    phdisp : ppy.PhdispCalcMode
+
+    """
+    yml_path = os.path.join("refs", "sto_spectral-cum.yml")
+    spectral_path = os.path.join("refs", "sto_spectral_cumulant.h5")
+    return ppy.SpectralCumulant.from_hdf5_yaml(spectral_path, yml_path)
+
```

Comment:
Please add just one more test that tests not the plot, but the SpectralCumulant object itself.

You could test just the shapes of the arrays, e.g.

```
    np.testing.assert_equal(obj.array1.shape, (181, 110))
    np.testing.assert_equal(obj.array2.shape, (181, 21))
```

Reply:
do you have an example in the current perturbopy code?

Reply:
Hey @yaoluo , I did't find the examples in the current perturbopy tests, but, if I un understand right, @imaliyov suggest the following idea: make class instance and then compare the size of the attributes with the reference one (or the values of this attributes with the reference one).